Assets/HoloToolkit/Input/Tests/Scripts/SimpleGridGenerator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class SimpleGridGenerator : MonoBehaviour
```

Comment:
With this file, were you trying to demonstrate that this works with dynamically created prefab instances?